crates/pop-cli/src/commands/clean.rs

```diff
@@ -54,38 +61,69 @@ impl<'a, CLI: Cli> CleanCacheCommand<'a, CLI> {
 			self.cache.to_str().expect("expected local cache is invalid")
 		))?;
 
-		// Prompt for selection of artifacts to be removed
-		let selected = {
-			let mut prompt =
-				self.cli.multiselect("Select the artifacts you wish to remove:").required(false);
-			for (name, path, size) in &contents {
-				prompt = prompt.item(path, name, format!("{}MiB", size / 1_048_576))
+		// if args.all flag is present, then remove all artifacts
+		if args.all {
+			// Display all artifacts to be deleted and get confirmation
+			let list = style(format!(
+				"> {}",
+				&contents
+					.iter()
+					.map(|(name, _, size)| format!("{} : {}MiB", name, size / 1_048_576))
+					.collect::<Vec<_>>()
+					.join(", ")
+			))
+			.dim()
+			.to_string();
+
+			if !confirm(format!(
+				"Would you like to cleanup all cache artifacts...\n {list}"))
+			.initial_value(true)
+			.interact()? {
+				outro_cancel("Failed to clean cache")?;
+				return Ok(());
 			}
-			prompt.interact()?
-		};
-		if selected.is_empty() {
-			self.cli.outro("ℹ️ No artifacts removed")?;
-			return Ok(());
-		};
 
-		// Confirm removal
-		let prompt = match selected.len() {
-			1 => "Are you sure you want to remove the selected artifact?".into(),
-			_ => format!(
-				"Are you sure you want to remove the {} selected artifacts?",
-				selected.len()
-			),
-		};
-		if !self.cli.confirm(prompt).interact()? {
-			self.cli.outro("ℹ️ No artifacts removed")?;
-			return Ok(());
-		}
+			for (_, file, _) in &contents {
+				// confirm removal
+				remove_file(&file)?;
+				self.cli.outro(format!("ℹ️ {} artifacts removed", contents.len()))?;
```

Comment:
This message appears n times, just move the message after the loop:
`ℹ️ 5 artifacts removed`